hack/run-all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
The purpose of these is to set the shell to "sane mode" and make it's execution easier to understand / work with, as it's rare that the behavior from these options isn't what you wanted in the first place.